website/src/index-page.tsx

```diff
@@ -1,119 +1,374 @@
+import { FiPlay, FiGithub } from 'react-icons/fi'
+
 import httpImage from '../public/assets/http.svg'
-import ecosystemImage from '../public/assets/ecosystem.svg'
 import subscriptionsImage from '../public/assets/subscriptions.svg'
 import uploadsImage from '../public/assets/uploads.svg'
-import yogaImage from '../public/assets/yogaHome.svg'
 
-export const ITEMS: IHeroIllustrationProps[] = [
-  {
-    title: 'GraphQL over HTTP compliant',
-    description: 'Yoga follows the GraphQL over HTTP specification.',
-    image: {
-      src: httpImage,
-      placeholder: 'empty',
-      alt: 'Server over HTTP',
-    },
-    link: {
-      children: 'Learn more',
-      title: 'Learn more',
-      href: '/docs',
-    },
-  },
-  {
-    title: 'Extensible GraphQL Engine powered by Envelop',
-    description:
-      'Add authentication, caching, error reporting or rate limiting with ease.',
-    image: {
-      src: ecosystemImage,
-      placeholder: 'empty',
-      alt: 'Ecosystem',
-    },
-    flipped: true,
-    link: {
-      children: 'Learn more',
-      title: 'Learn more',
-      href: '/docs/features/envelop-plugins',
-    },
-  },
-  {
-    title: 'GraphQL Subscriptions over HTTP',
-    description:
-      'Run GraphQL Subscriptions over HTTP instead of WebSockets via Server Sent Events.',
-    image: {
-      src: subscriptionsImage,
-      placeholder: 'empty',
-      alt: 'Subscriptions',
-    },
-    link: {
-      children: 'Learn more',
-      title: 'Learn more',
-      href: '/docs/features/subscriptions',
-    },
-  },
-  {
-    title: 'Handle file uploads with GraphQL',
-    description:
-      'Enable file uploads using GraphQL mutations with no additional packages needed.',
-    image: {
-      src: uploadsImage,
-      placeholder: 'empty',
-      alt: 'Uploads',
-    },
-    flipped: true,
-    link: {
-      children: 'Learn more',
-      title: 'Learn more',
-      href: '/docs/features/file-uploads',
-    },
-  },
+const gradients: [string, string][] = [
+  ['#8b5cf6', '#6d28d9'], // violet
+  ['#06b6d4', '#0e7490'], // cyan
+  ['#f59e0b', '#d97706'], // amber
 ]
-import Link from 'next/link'
 
-export default function IndexPage() {
+function pickGradient(i: number) {
+  const gradient = gradients[i % gradients.length]
+  if (!gradient) {
+    throw new Error('No gradient found')
+  }
+  return gradient
+}
+
+export function IndexPage(): ReactElement {
+  return (
+    <div className="flex flex-col">
+      <FeatureWrapper>
+        <div className="w-full">
+          <div className="py-20 sm:py-24 lg:py-32 my-6">
+            <h1
+              className="max-w-screen-md mx-auto font-extrabold text-5xl sm:text-5xl lg:text-6xl text-center bg-gradient-to-r from-fuchsia-400 to-fuchsia-700 dark:from-fuchsia-400 dark:to-fuchsia-700 bg-clip-text text-transparent"
+              style={{
+                // TODO: default line height is 1, but that cuts of the "g" in the title
+                lineHeight: 1.2,
```

Comment:
Use tailwind CSS class instead ?

Reply:
`text-6xl` class has the line-height set, adding `leading-tight` doesn't work.